AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/ReconcileAndLocalSave/ModelReconciliationQueue.swift

```diff
@@ -14,6 +14,7 @@ enum ModelReconciliationQueueEvent {
     case paused
     case connected(String)
     case mutationEvent(MutationEvent)
+    case mutationEventDropped(String)
```

Comment:
can these have named associated types or typealiases? is it `connected(ModelName)`? `connected(modelName: String)`? `mutationEventDropped(modelName: String)`?

Reply:
I updated as `mutationEventDropped(modelName: String)`